Zeeguu Reader/Zeeguu Reader/ArticleListViewController.swift

```diff
@@ -116,26 +116,17 @@ class ArticleListViewController: ZGTableViewController {
 	}
 
 	override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-		let queueCell = tableView.dequeueReusableCellWithIdentifier("Cell")
-		var cell: UITableViewCell
+		let queueCell = tableView.dequeueReusableCellWithIdentifier("Cell") as? ArticleTableViewCell
+		
+		let article = articles[indexPath.row]
```

Comment:
This is a method that is called by the UITableViewController. As it is part of a protocol made by Apple, it is all in the Apple docs, as well as in Xcode in the right panel or by Option+click.

Link to Apple docs:
https://developer.apple.com/library/prerelease/ios/documentation/UIKit/Reference/UITableViewDataSource_Protocol/index.html#//apple_ref/occ/intfm/UITableViewDataSource/tableView:cellForRowAtIndexPath: